README.md

```diff
@@ -68,4 +68,96 @@ By default all image uploads are uploaded to a cloud provider of your choice. I
 
 ## Installation
 
-Coming soon...
+### Server
+
+* Clone this repo!
+* Download and install https://getcomposer.org/download/
+
+```
+php composer.phar install
+```
+
+* Follow instructions @ https://laravel.com/docs/5.7/homestead to setup homestead
+* Edit `Homestead.yaml`, map the knot-server folder, change the hostname of the server to knot-server.test
+* In `/etc/hosts` map knot-server.test to the ip listed in Homestead.yaml
+
+```
+vagrant up
+```
+
+* Login to server: `vagrant ssh`
+* Setup database user + db
+
+```
+mysql
+
+> CREATE USER 'forge'@'localhost' IDENTIFIED BY '';
+> GRANT ALL PRIVILEGES ON * . * TO 'forge'@'localhost';
+> FLUSH PRIVILEGES;
+> CREATE DATABASE forge;
+```
+
+* Run migrations and install dependencies
+
+    php artisan migrate
+      php artisan passport:install
```

Comment:
After `passport:install` you also need to run

```
php artisan passport:client --password
```

to create the password client.

Reply:
hrmm, I had trouble with this, then looked in the DB and saw two existing client entries in the `oauth_clients` table, and everything seemed to work after using the `Knot Password Grant Client `

Reply:
Yeah `php artisan passport:client --password` creates the `Knot Password Grant Client`. Are you saying there was some other client entry already in your `oauth_clients` table?

Reply:
I think so?

I'll start from scratch and go through these instructions to see how it works out, its totally possible that I ran it and forgot I did

Reply:
I've been running it off Valet, so I'll try and get it running on a Homestead box to confirm 😄